### ISSUE TITLE
download broken

### ISSUE THREAD

**Issue by MisterQuiggly** [HakkaTjakka/MinecraftWorldEditor#4]
https://drive.google.com/drive/folders/1RuIClj5k9hI7i3NJv5cLOhdohfn4S9tR
your download link doesn't work.

**Comment by HakkaTjakka** [comment]
Yes, sorry for my late reply. Yes indeed. I had an account for $10 a month but Google started to block files and stuff because of people marked it as 'proprietary content' so i quit with using drive.
For the BTE121 build of The Nederlands (conversion Google Earth to minecraft) i'm hosting now my own web server on  http://84.104.196.249/ with the Cubic Chunks version of the build. I've lost the normal (stacked, normal 1.12.2 version), and Github also deleted the BTE121 project of The Netherlands.
Its a multi terrabyte archive. 
This repo here has lots of different code all in one .exe, but there is some manual of it.
I saw others are using also parts of this code like the voxelizer and read/writing of .mca files for minecraft, and the downloading from Google Earth 3d data, converting it etc. Some use it for the BTE121 project.
Mainly this is 'junk-code', unreadable for noobs, but working... and accurate etc. And published to share (parts of) the code for others, like difficult mathematical stuff like 3d conversion rendering manipulation etc. Video editing. Shaders. Gaming, c/c++ etc.

**Comment by MisterQuiggly** [comment]
Haha, I think whatever I was specifically trying to accomplish I found another solution for but thanks for the reply nonetheless!

**Comment by HakkaTjakka** [comment]
When online:

http://84.104.196.249/Storage/REPO/
http://84.104.196.249/Storage/EXTENSION/REPO/ (also .zip per region file in Cubic Chunks Format)

Also on ftp...
If dir EXTENSION is empty, then the internal hard drive is not mounted, somehow. Then i have to mount it manually.
(doesn't mount on startup somehow... don't know how to fix, ubuntu Linux, is in fstab but doesn't do anything on startup)

.rar files whole of Holland: 525.2 GigaByte.
.zip files on REPO2:  213,137 .zip files 551.03 GigaByte.

Shortcut
http://84.104.196.249/REPO/
http://84.104.196.249/REPO2/
